core/opendaq/streaming/include/opendaq/mirrored_signal_impl.h

```diff
@@ -581,46 +581,33 @@ ErrCode MirroredSignalBase<Interfaces...>::getOnUnsubscribeComplete(IEvent** eve
 template <typename... Interfaces>
 EventPacketPtr MirroredSignalBase<Interfaces...>::createDataDescriptorChangedEventPacket()
 {
+    const EventPacketPtr eventPacketFromConfig = Super::createDataDescriptorChangedEventPacket();
+    const auto params = eventPacketFromConfig.getParameters();
```

Comment:
Do we need to ask the config client for the descriptor if we already have a descriptor assigned?